projects/angular-image-viewer/package.json

```diff
@@ -1,20 +1,21 @@
 {
-  "name": "@clarivate/angular-image-viewer",
-  "version": "1.2.6-ng7",
+  "name": "@clarivate/angular-x-image-viewer",
```

Comment:
this will change the namespace that the module is published, is this intentional?